docs/SNAPTEL.md

```diff
@@ -18,16 +18,48 @@ limitations under the License.
 -->
 
 # snaptel
-A powerful telemetry framework
 
-## Usage
-The `snap-telemetry` package installs `snaptel` in `/usr/local/sbin/snaptel`, and `snapteld` in `/usr/local/sbin/snapteld`. Either ensure `/usr/local/bin:/usr/local/sbin` is in your path, or use fully qualified filepath to the `snaptel`/`snapteld` binary:
+The command-line interface (CLI) for Snap - a powerful telemetry framework
+
+## General information
+
+ - **Snap CLI on top of [REST API V1](REST_API_V1.md)**
+
+    The `snaptel` which bases on REST API V1 is available in the main Snap repo, as a part of
+[Snap releases](https://github.com/intelsdi-x/snap/releases) and `snap-telemetry` package.
+
+
+ - **Snap CLI on top of [REST API V2](REST_API_V2.md)**
+
+    The _alpha version_ of `snaptel` which bases on REST API V2 is available in separated repo
+[snap-cli](https://github.com/intelsdi-x/snap-cli).
+
+    The separated Snap API client and CLI support [Swagger Open API Specification](https://swagger.io/specification). To get more help on the Snap API client and Snap CLI check out:
```

Comment:
This section talks about "separated Snap API client" which is related yet different from Snap CLI - maybe consider a small sub-section explaining new Snap API client, or just putting link where it is described? Maybe a bit of introduction saying that new CLI is based on new Snap API client which in turn is talking to API v2 would be good?